lib/scanner/index.js

```diff
@@ -81,30 +77,22 @@ class Scanner extends Emitter {
    * @private
    */
   onItemFound(item) {
-    var url = item.url;
+    var shouldPopulate = this.shouldPopulateItem(item.url);
```

Comment:
Is there value in this if we're currently the only consumer of this module? It's hard to know when to stop. If this we're a standalone library used by other projects, or even just used in more than one place in our project, I'd understand.